SPECS/python-rich/python-rich.spec

```diff
@@ -0,0 +1,175 @@
+Name:           python-rich
+Version:        13.7.1
+Release:        1%{?dist}
```

Comment:
Release should be bumped to distinguish it from previous fedora entry in the changelog.